.env.example

```
@@ -0,0 +1,32 @@
+# Docker Compose Environment Variables
+# Copy this file to .env and configure for your environment
+
+# --- Docker Image Tags ---
+# Using single repository (monkscode/nlrf) with descriptive tags
+# Options: 
+#   - fastapi-latest / browser-service-latest (stable from main)
+#   - fastapi-develop / browser-service-develop (latest from develop)
+#   - fastapi-1001 / browser-service-1001 (specific version)
+FASTAPI_IMAGE_TAG=monkscode/nlrf:fastapi-pr-50
+BROWSER_SERVICE_IMAGE_TAG=monkscode/nlrf:browser-service-pr-50
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BROWSER_SERVICE_IMAGE_TAG key should go before the FASTAPI_IMAGE_TAG key

(UnorderedKey)